modules/unpickler/src/test/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Scala3UnpicklerTests.scala

```diff
@@ -293,6 +293,26 @@ abstract class Scala3UnpicklerTests(val scalaVersion: ScalaVersion) extends FunS
     debuggee.assertFormat("example.B", javaSig("java.lang.String"), "B.m(): String")
   }
 
+  test("find outter field") {
+    val source =
+      """|package example
+         |class A :
+         |  private val x =2 
+         |  class B : 
+         |    class C :
+         |      private val y = x+2
+         |""".stripMargin
+    val debuggee = TestingDebuggee.mainClass(source, "example.Main", scalaVersion)
+
+    debuggee.assertFormatAndKind(
+      "example.A$B$C",
+      "example.A$B example$A$B$C$$$outer()",
+      "A.B.C.<outer A>",
```

Comment:
This is the outer field in C that returns a B

```suggestion
      "A.B.C.<outer>: B",
```